Dockerfile

```diff
@@ -1,7 +1,16 @@
-ARG PHP_VERSION
+ARG PHP_VERSION=8.3
```

Comment:
I had to define default value because without it, it wasn't possible to build `sphinx-lint` target:

```
failed to solve: failed to parse stage name "php:-cli-alpine3.18": invalid reference format
```